spec/controllers/api/v1/rents_controller_spec.rb

```diff
@@ -41,6 +41,10 @@
       it 'responds with 201 status' do
         expect(response).to have_http_status(:created)
       end
+
+      it 'plus rent count in user' do
+        expect(user.reload.rents_count).to equal(1)
```

Comment:
This test is okay, but I think we can improve using this:
https://relishapp.com/rspec/rspec-expectations/v/2-0/docs/matchers/expect-change